### DISCUSSION TITLE
Integration wrong in ln and 1/x (Why is there no Ln function)?

### DISCUSSION THREAD

**Discussion by xwaeaewcrhomesysplug** [axkr/symja_android_library#270] [Q&A] [answered]
**Describe the bug**
Integration 1/x basic propeties gives wrong output.

**To Reproduce**

Example input Symja input expression:

```mma
>> 
D(Ln(x),x)
1/x
Integrate(1/x,x)
Log(x)
```

Error result:

```mma
Integrate(1/x,x)
Log(x)
```

**Expected behavior**

Expected (correct) result expression:

```mma
Integrate(1/x,x)
Ln(x)
```

**Java**
Java15,win10

**Additional context**
other integration test ok.

**Comment by tranleduy2000** [comment, marked as answer]
`Log(x)` is `Ln(x)`

## Log

```
Log(z)
```

> returns the natural logarithm of `z`.

```
Log(base, z)
```
> get the logarithm of `z` for base `base`.

**Comment by axkr** [comment]
We are using the same conventions as Mathematica for the `Log` function:
- https://reference.wolfram.com/language/ref/Log.html

So there is only a `Log` function for the natural logarithm.
The "alias" function `Ln(x)` is only rewritten as `Log(x)`
 
Note that `Log(a,b)` is converted to `Log(b)/Log(a)`
